virttest/utils_libvirt/libvirt_numa.py

```diff
@@ -131,6 +132,8 @@ def _get_first_continuous_numa_node_index(node_list):
             numa_nodeset = '%s-%s,^%s' % (str(node_list[candidate_index]),
                                           str(node_list[candidate_index + 1]),
                                           str(node_list[candidate_index + 1]))
+    elif ignore_error:
+        LOG.error("Supported formats are not found. No parsing happens.")
```

Comment:
The function will return `numa_nodeset` as it was input when `ignore_error` is True, right?

Reply:
@chloerh , yes, you are right. That's what I want.

Reply:
OK.